python/cql2.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+from os import PathLike
+
+class SqlQuery:
+    query: str
+    params: list[str]
+
+class Expr:
```

Comment:
When we come back to docs: there are pros and cons between writing the docstrings in the rust code or in the pyi file

Reply:
Agreed, https://github.com/developmentseed/cql2-rs/issues/31.